src/test/java/com/bathanh/apibook/domain/auths/JwtTokenServiceTest.java

```diff
@@ -1,35 +1,61 @@
 package com.bathanh.apibook.domain.auths;
 
 import com.bathanh.apibook.properties.JwtProperties;
+import io.jsonwebtoken.Claims;
+import io.jsonwebtoken.Jwts;
 import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
 import org.junit.jupiter.api.extension.ExtendWith;
-import org.mockito.InjectMocks;
 import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
 import org.mockito.junit.jupiter.MockitoExtension;
+import org.springframework.security.core.authority.SimpleGrantedAuthority;
+
+import java.util.Collections;
+import java.util.Date;
+
+import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
+import static org.mockito.Mockito.when;
 
 @ExtendWith(MockitoExtension.class)
-class JwtTokenServiceTest {
+public class JwtTokenServiceTest {
```

Comment:
class, method trong unit test thì đừng để public. Cứ để trống vậy

```suggestion
class JwtTokenServiceTest {
```